### PR TITLE
Add global rules

### PR DESCRIPTION
**Pull request by fabxc**
Implement a GlobalRules resource which evalutes rules without any
scoping

### CODE REVIEW

File: pkg/operator/apis/monitoring/v1alpha1/types.go [hunk 1]
```diff
@@ -1156,6 +1156,36 @@ type ClusterRulesList struct {
 	Items           []ClusterRules `json:"items"`
 }
 
+// GlobalRules defines Prometheus alerting and recording rules that are scoped
+// to all data in the queried project. Only metric data from the current cluster is processed
```
Comment (pintohutch) [inline pkg/operator/apis/monitoring/v1alpha1/types.go:1160]: The godoc needs to be updated; this is a description of `ClusterRules` behavior.

**Review by pintohutch — APPROVED** [review]
LGTM % nits

File: pkg/operator/apis/monitoring/v1alpha1/types.go (hunk 1 above)
Comment (fabxc) [reply pkg/operator/apis/monitoring/v1alpha1/types.go:1160]: No :) it is adapted.

Comment (fabxc) [reply pkg/operator/apis/monitoring/v1alpha1/types.go:1160]: Ah, actually forgot to delete one old sentence. Fixed.

Comment (pintohutch) [reply pkg/operator/apis/monitoring/v1alpha1/types.go:1160]: So I'm a bit hung up on
```
Only metric data from the current cluster is processed
```
Shouldn't this be something like
```
Only metric data from the queried project is processed
```

Comment (fabxc) [reply pkg/operator/apis/monitoring/v1alpha1/types.go:1160]: Yes, that's the old sentence I meant. Didn't push anymore yesterday after the presubmit run.

**Pull request merged**